kong/conf_loader.lua

```diff
@@ -587,6 +608,25 @@ local function load(path, custom_conf)
     end
   end
 
+  local header_tokens_clone = tablex.deepcopy(header_tokens)
+  -- load headers configuration
+  for _, token in ipairs(conf.headers) do
+    if token ~= "off" then
+      header_tokens_clone[token] = true
```

Comment:
I think we should allow for case-insensitive values when specifying headers in this directive (just like HTTP headers are). E.g.:
```
  headers = latency_tokens, Via
is same as:
  headers = latency_tokens, via
```